packages/php-table-backend-utils/src/Connection/Bigquery/BigQueryClientWrapper.php

```diff
@@ -75,9 +86,27 @@ public function runJob(JobConfigurationInterface $config, array $options = []):
         assert($options['backOffPolicy'] instanceof BackOffPolicyInterface);
         $retryPolicy = new SimpleRetryPolicy($options['retryCount']);
         $proxy = new RetryProxy($retryPolicy, $options['backOffPolicy']);
-        $job = $proxy->call(function () use ($config, $options): Job {
-            return $this->startJob($config, $options);
-        });
+        try {
+            $job = $proxy->call(function () use ($config, $options): Job {
+                return $this->startJob($config, $options);
+            });
+        } catch (Throwable $e) {
+            $jobConfig = $config->toArray();
+            $this->logger->warning('BigQuery job failed to start.', [
+                'exception' => $e,
+                'jobConfig' => $jobConfig,
+            ]);
+            if (str_contains($e->getMessage(), 'Already Exists: Job')) {
```

Comment:
Neslo by to osetrit na ten kod 409 aspon pripadne neverati to toto ? https://github.com/googleapis/google-cloud-php/blob/main/Core/src/RequestWrapper.php#L414 (ConflictException)